app/Http/Controllers/Api/CTS/ValidationsController.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Http\Controllers\Api\CTS;
+
+use App\Repositories\Cts\ValidationPositionRepository;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+
+class ValidationsController
+{
+    public function view(Request $request)
+    {
+        if (!$request->get('position')) {
+            return response()->json([
+                'status'  => '400',
+                'message' => 'No position was supplied.'
+            ])->setStatusCode(400);
```

Comment:
Status code can just be the second paramter to the json method.

`public function json($data = [], $status = 200, array $headers = [], $options = 0);`
REF:  `Illuminate\Contracts\Routing\ResponseFactory`

Reply:
Part of me kind of likes how explicit this is, but any decent IDE will be telling you the status is the second para - resolved in be653e94